libcaf_core/caf/detail/actor_system_access.hpp

```diff
@@ -28,6 +30,8 @@ class CAF_CORE_EXPORT actor_system_access {
 
   detail::mailbox_factory* mailbox_factory();
 
+  detail::daemons* daemons();
+
```

Comment:
Doesn't adding this method break ABI?

Reply:
Adding functions is fine - except when they're virtual. Generally, anything that breaks memory layout (touch member variables or virtual member functions) or changes existing functions is an issue. Adding more stuff is fine. Of course you can't build against, say 1.2, and then expect your binary to work with 1.1. Older version simply might not have certain symbols. But anything >= 1.2 should work (as long it's the same major version).

Reply:
OK, got a bit mixed up here, didn't know that non-virtual member functions are fine. Thanks.